### PR TITLE
Added openid ACR parameter information.

### PR DESCRIPTION
**Pull request by mrs**
!!!!! STOP AND READ !!!!!

If the dropdown above says "base fork: lord/master", you are submitting your change to ALL USERS OF SLATE, not just your company. This is probably not what you want. Click "base fork" to change it to the right place.

If you're actually trying to submit a change to upstream Slate, please submit to our dev branch, PRs sent to the master branch are generally rejected.

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -208,29 +212,30 @@ Now, the token endpoint will also return an `id_token` param containing a [JWS](
...
 
-Claim     | Description
---------- | -----------
-iat       | The time at which the id_token was issued
-exp       | The expiration time on or after which the id_token will be rejected expressed as a [NumericDate](#terminology)
+Claim | Description
+----- | -----------
+iat   | The time at which the id_token was issued
+exp   | The expiration time on or after which the id_token will be rejected expressed as a [NumericDate](#terminology)
+acr   | The Level of Assurance to which the user is currently authenticated. Only the SMS authentication method will currently result in a less than "http://eidas.europa.eu/LoA/high" LoA.
```
Comment (kimat) [inline source/index.html.md]: will not return "http://eidas.europa.eu/LoA/high" but "http://eidas.europa.eu/LoA/substential"

Comment (kimat) [inline source/index.html.md]: ~~ The Level of Assurance to which the user is currently authenticated ~~

Comment (mrs) [inline source/index.html.md]: What do you mean ? You want to remove the definition of the parameter ?

Comment (mrs) [inline source/index.html.md]: Then we can't change it to "low" later on, and for the time being the only thing that matters is that it isn't "high". My version reflects that and is more flexible than your proposition.

**Pull request merged**